motoko/hello-angular-motoko/README.md

```diff
@@ -0,0 +1,82 @@
+# hello-angular-motoko
+This is an example of locally implementing and deploying the Internet Computer's motoko project (0.10.1) along side Angular 14.
+
+-----------
+## Getting started
+
+- [Install DFX](https://sdk.dfinity.org/docs/quickstart/local-quickstart.html). Please keep in mind the dfx cli currently only runs on Linux and Apple based PCs.
```

Comment:
```suggestion
- [Install DFX](https://internetcomputer.org/docs/current/developer-docs/quickstart/local-quickstart).
```
Updating with the current link, removing system requirements because that is a future stale bit of information.